style.css

```diff
@@ -1 +1,267 @@
-/*  Add your styling here */
+/* General rules */
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body {
+    font-style: italic;
```

Comment:
By adding this rule here, you are making a choice to use italic text globally in your page. This can cause the following issue:
Some of the html tags like `<em>` and `<i>` will be applying italic styling by default when used to add emphasis on the text. This means that if you use italics everywhere on your page you lose the capability of emphasising text in your document, which is bad for accessibility reasons. A way to solve this is to remove the rule from body and apply it to more specific selectors or apply different styles to `em` and `i` selector to differentiate them. Eg `em,i { font-style: bold; }`    See this for more detail: https://css-tricks.com/how-to-italicize-text/

Reply:
I am supposed to think of the accessibility more.  Deleted  italic font style on body.